WBooks/Views/BookIndex/BookIndexViewModel.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  BookIndexViewModel.swift
+//  WBooks
+//
+//  Created by Diego Quirós on 3/13/18.
+//  Copyright © 2018 Wolox. All rights reserved.
+//
+
+import UIKit
+import ReactiveSwift
+
+internal final class BookIndexViewModel {
+
+    var bookViewModels: MutableProperty<[BookViewModel]> = MutableProperty([])
+    
+    private let _bookRepository: BookRepositoryType
+    
+    init(bookRepository: BookRepositoryType) {
+        
+        _bookRepository = bookRepository
+        
+//        bookViewModels <~ bookRepository.fetchBooks()
+//            .map {
+//                print(">>>>>>> \($0)")
+//                return $0.map { book in
+//                    print(">>>>>>>>>>>>>>>>>>>>> FOO")
+//                    return BookViewModel(book: book)
+//                }
+//            }
+//            .liftError()
+        
+        bookRepository.fetchBooks()
```

Comment:
I would suggest to avoid performing request when you initialize an object. It is better to make that explicit via a method exposed by the view model.